### DISCUSSION TITLE
get rdt

### DISCUSSION THREAD

**Discussion by Kepler-Elliott** [amzn/selling-partner-api-models#4378] [Q&A]
Hello, when I got the rdt, I filled path with  /reports/2021-06-30/documents/"+reportDocumentId, But what he returns is this {"errors":[{"code":"InvalidInput","details":"","message":"Invalid or unsupported document ID found"}]} Why

**Comment by weilinggu** [comment]
Hi @Kepler-Elliott,

I responded to your question in the other [discussion](https://github.com/amzn/selling-partner-api-models/discussions/4334#discussioncomment-11488882)
Are you following the tutorial [here](https://developer-docs.amazon.com/sp-api/docs/reports-api-v2021-06-30-tutorial-request-a-report) on the documentation site on requesting a report?

Thanks,
Weiling
Amazon Selling Partner Services
